velox/exec/TableWriter.cpp

```diff
@@ -53,8 +52,7 @@ TableWriter::TableWriter(
   }
 
   mappedType_ = ROW(std::move(names), std::move(types));
-  writeProtocol_ = connector::WriteProtocol::getWriteProtocol(
-      tableWriteNode->commitStrategy());
+  commitStrategy_ = tableWriteNode->commitStrategy();
```

Comment:
nit: consider making commitStrategy_ const and initialize it using initialization list

```
insertTableHandle_(
          tableWriteNode->insertTableHandle()->connectorInsertTableHandle()),
commitStrategy_(tableWriteNode->commitStrategy())
```

Reply:
Revised this and others